java/templates/jolokia-opts

```diff
@@ -1,8 +1,5 @@
 #!/bin/sh
```

Comment:
I'm a bit confused, why jolokia-opts is included in the template directory as its origin is from `agent-bond` where it is [maintained](https://github.com/fabric8io/agent-bond/blob/master/fish-pepper/jolokia/fp-files/jolokia-opts) and then included via the block in [fish-pepper.yml](https://github.com/fabric8io-images/s2i/blob/master/java/fish-pepper.yml#L16). 

To resolve this, one of the following should be done:

* Either remove the block reference in fish-pepper.yml and remove the reference to jolokia.opts in the [Dockerfile template](https://github.com/fabric8io-images/s2i/blob/master/java/templates/Dockerfile#L46-L49). 
* Or update [jolokia-opts](https://github.com/fabric8io/agent-bond/blob/master/fish-pepper/jolokia/fp-files/jolokia-opts) in agent-bond to be more universal, allowing to write out the property to a file as done here.

I'd prefer the latter option has it then could be reused by other images as well. Also, I would simply prepare the argument string and split it on ',' or the other way round: Create a line-by-line string and split on new line (probably safer w.r.t to escaped commas e.g. in the cert's subject name).

btw, I'm not sure why it works currently, but I guess the template files are copied over the files coming from the external block later.